README.md

```diff
@@ -1,120 +1,422 @@
+
+```jsx
+<ix-img
+  src="image.jpg"
+  width="100" // This width and the height below sets what resolution the component should load from the CDN and the size of the resulting image
+  height="200"
+  fixed
+></ix-img>
+```
+
+This will generate an image element like:
+
+<!-- prettier-ignore-start -->
+```jsx
+<img 
+  src="https://.../image.jpg?w=100&h=200" // Notice the w and h parameters here
   srcset="
-    https://assets.imgix.net/unsplash/motorbike.jpg?w=500&dpr=1&q=75&ixlib=ng-0.0.1 1x,
-    https://assets.imgix.net/unsplash/motorbike.jpg?w=500&dpr=2&q=50&ixlib=ng-0.0.1 2x,
-    https://assets.imgix.net/unsplash/motorbike.jpg?w=500&dpr=3&q=35&ixlib=ng-0.0.1 3x,
-    https://assets.imgix.net/unsplash/motorbike.jpg?w=500&dpr=4&q=23&ixlib=ng-0.0.1 4x,
-    https://assets.imgix.net/unsplash/motorbike.jpg?w=500&dpr=5&q=20&ixlib=ng-0.0.1 5x
-  "
+    image.jpg?w=100&h=200&dpr=1 1x, 
+    image.jpg?w=100&h=200&dpr=2 2x, ..." // This allows the image to respond to different device DPRs
+  width="100" 
+  height="200" 
 />
 ```
+<!-- prettier-ignore-end -->
 
-The `ixSrcset` attribute accepts modifier options, providing users the ability to fine-tune the `srcset` generated. See [imgix-core-js](https://github.com/imgix/imgix-core-js#srcset-generation) documentation for details on all available modifiers.
+Fixed image rendering will automatically append a variable `q` parameter mapped to each `dpr` parameter when generating a srcset. This technique is commonly used to compensate for the increased filesize of high-DPR images. Since high-DPR images are displayed at a higher pixel density on devices, image quality can be lowered to reduce overall filesize without sacrificing perceived visual quality. For more information and examples of this technique in action, see [this blog post](https://blog.imgix.com/2016/03/30/dpr-quality).
+This behavior will respect any overriding `q` value passed in via `imgixParams` and can be disabled altogether with the boolean property `disableVariableQuality`.
 
 ```html
-<img
-  src="https://assets.imgix.net/unsplash/motorbike.jpg"
-  [ixSrcset]="{ minWidth: 800, maxWidth: 1800 }"
-/>
+<ix-img src="image.jpg" width="100" disableVariableQuality ></ix-img>
 ```
 
-Generates a more tailored attribute:
+will generate the following srcset:
+
+```html
+https://.../image.jpg?q=75&w=100&dpr=1 1x,
+https://.../image.jpg?q=50&w=100&dpr=2 2x,
+https://.../image.jpg?q=35&w=100&dpr=3 3x,
+https://.../image.jpg?q=23&w=100&dpr=4 4x,
+https://.../image.jpg?q=20&w=100&dpr=5 5x
+```
+
+#### Lazy-Loading
+
+For lazy loading, there are a few options to choose from. They're listed here, along with our recommendation at the end.
+
+1. Native lazy-loading with `loading="lazy"`. As of May 2020, this is shipped in stable versions of Chrome and Firefox. [Example](#lazy-loading-native)
```

Comment:
> [Example](#lazy-loading-native)
Looks like this section is missing from the README

Reply:
👀